modules/nf-core/gatk4/genomicsdbimport/main.nf

```diff
@@ -16,7 +16,7 @@ process GATK4_GENOMICSDBIMPORT {
     output:
     tuple val(meta), path("$prefix")        , optional:true, emit: genomicsdb
     tuple val(meta), path("$updated_db")    , optional:true, emit: updatedb
-    tuple val(meta), path("*.interval_list"), optional:true, emit: intervallist
+    tuple val(meta), path("$interval_list"), optional:true, emit: intervallist
```

Comment:
heads up: these changes should be made in nf-core/modules and then imported here so others benefit from the work too :)

Reply:
I have opened an issue (https://github.com/nf-core/modules/issues/8393) and PR to nf-core modules (https://github.com/nf-core/modules/pull/8394).